gensim/models/word2vec.py

```diff
@@ -1986,9 +1986,6 @@ def _load_specials(self, *args, **kwargs):
             for a in ('hashfxn', 'layer1_size', 'seed', 'syn1neg', 'syn1'):
                 if hasattr(self.trainables, a):
                     setattr(self, a, getattr(self.trainables, a))
-            if hasattr(self, 'syn1'):
-                self.syn1 = self.syn1
```

Comment:
The actual intent here was to move `self.trainables.syn1` back to `self.syn1` to undo the `.trainables` complexification. So I don't think *just* deleting will fix user's raised issue. May need to be:

    if hasattr(self.trainables, 'syn1'):
        self.syn1 = self.trainables.syn1

There's then no need to `del` anything in this branch, as the whole `self.trainables` is dropped outside on the next line.

Reply:
That's already done in the loop above.

Reply:
OK, so perhaps there's more we can do in the test, because it should fail (not pass, like it does currently). What do we need to do?

Reply:
I think the test should pass now. Why do you think it should fail?

Reply:
Because of this:

> So I don't think just deleting will fix user's raised issue. 

But after re-reading the thread, it seems like the above comment is resolved, so we're good to go, right @gojomo ?

Reply:
Aha, I'd missed the prior attribute move-over. If it fixes the user problem, looks good! But made a separate coment about the details of the new test.